assembly-syntax/src/ast/instruction/mod.rs

```diff
@@ -218,14 +220,22 @@ pub enum Instruction {
     MemLoadImm(ImmU32),
     MemLoadW,
     MemLoadWImm(ImmU32),
+    MemLoadWLe,
+    MemLoadWLeImm(ImmU32),
+    MemLoadWBe,
+    MemLoadWBeImm(ImmU32),
```

Comment:
Do we still need to keep `MemLoadW` and `MemLoadWImm`? I thought these would be replaced by the `Be` variants.